yarn.lock

```diff
@@ -6402,7 +6402,7 @@ mkdirp@^0.5.0, mkdirp@^0.5.1, mkdirp@^0.5.3, mkdirp@^0.5.5:
   dependencies:
     minimist "^1.2.5"
 
-mocha@^8.2.0, mocha@^8.2.1:
+mocha@^8.2.0:
```

Comment:
Should mocha still be in here at all?

Reply:
Yes, Mocha is still used to test all the bedrock functionality.